### PR TITLE
release `1.8.0`

### PR DESCRIPTION
**Pull request by zacharyburnett**
release `1.8.0` will be ready when the PRs for Build 9.0 (https://github.com/spacetelescope/jwst/milestone/45) are merged

### CODE REVIEW

**Comment by nden** [comment]
A couple minor comments inline. More importantly we need to constrain stcal on the right so that backwards incompatible versions do not get downloaded with  jwst 1.8.0

File: requirements-sdp.txt [hunk 1]
```diff
@@ -15,3 +15,79 @@
...
+pytest-openfiles==0.5.0
+python-dateutil==2.8.2
+PyWavelets==1.4.1
+PyYAML==6.0
+requests==2.28.1
+requests-mock==1.10.0
+scikit-image==0.19.3
+scipy==1.9.1
+semantic-version==2.10.0
+six==1.16.0
```
[the remark this answers is not in the document]
Comment (nden) [reply requirements-sdp.txt:81]: not critical for the release, just curious

File: requirements-sdp.txt [hunk 2]
```diff
@@ -15,3 +15,79 @@
...
+fonttools==4.37.4
+future==0.18.2
+getch==1.0
+gwcs==0.18.2
+idna==3.4
+imageio==2.22.1
+iniconfig==1.1.1
+jmespath==1.0.1
+jsonschema==4.9.1
+-e git+https://github.com/spacetelescope/jwst@1.8.0#egg=jwst
```
[the remark this answers is not in the document]
Comment (zacharyburnett) [reply requirements-sdp.txt:48]: Usually it is the commit hash:
https://github.com/spacetelescope/jwst/blob/943437a3e05bf37e557ecb3e6c3b887281be360b/requirements-sdp.txt#L46
but here instead I used the `1.8.0` tag to make sure that the installation uses the release commit

File: requirements-sdp.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (zacharyburnett) [reply requirements-sdp.txt:81]: from `pipdeptree`, it looks like it is required by `requests_mock` and `jwst` (via `poppy` and `matplotlib`):
```shell
pipdeptree --reverse --packages six
```
```
six==1.16.0e --reverse --packages six                    
  - python-dateutil==2.8.2 [requires: six>=1.5]
    - matplotlib==3.6.0 [requires: python-dateutil>=2.7]
      - BayesicFitting==3.0.1 [requires: matplotlib>=2.0]
        - jwst==1.7.3.dev24+gf7ecd39a [requires: BayesicFitting>=3.0.1]
      - poppy==1.0.3 [requires: matplotlib>=3.2.0]
        - jwst==1.7.3.dev24+gf7ecd39a [requires: poppy>=1.0.2]
  - requests-mock==1.10.0 [requires: six]
```

**Pull request merged**